README.md

```diff
@@ -45,6 +45,7 @@ A curated list of anything remotely related to mental health, sorted by alphabet
 ### Lists
 * [Mental Health Knowledge Base](https://mhwkb.github.io/) - List of resources about Mental Health.
 * [Selfcare.tech](http://selfcare.tech/) - List of resources mainly focussed on people in the tech industry, but really anyone can find something in here.
+* [Awesome-Autism Resource List](https://github.com/aspergirl-git/awesome-autism) -  A curated list of awesome resources for Autism Spectrum Disorder and Asperger's Syndrome.
```

Comment:
Thanks for the suggestion @aspergirl-git . This just needs to be moved to the top, as our goal is an alphabetized list.